src/store/modules/profile.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+export const GET_PROFILE_ERROR = 'GET_PROFILE_ERROR';
+export const GET_PROFILE_SUCCESS = 'GET_PROFILE_SUCCESS';
+
+const url =
+  'https://zinnia-ah-backend-staging.herokuapp.com/api/v1/users/profiles';
+
+export const getUserProfileError = error => ({
+  type: GET_PROFILE_ERROR,
+  error,
+});
+
+export const getUserProfile = profile => ({
+  type: GET_PROFILE_SUCCESS,
+  payload: profile,
+});
+
+export const getUserProfileRequest = () => async dispatch => {
+  try {
+    // const token = localStorage.getItem('token');
+    const token =
```

Comment:
We should not hardcode token as we are doing here.

Reply:
done